drivers/wifi/nxp/nxp_wifi_drv.c

```diff
@@ -1374,6 +1374,25 @@ static int nxp_wifi_set_twt(const struct device *dev, struct wifi_twt_params *pa
 
 	return ret;
 }
+
+static int nxp_wifi_set_btwt(const struct device *dev, struct wifi_twt_params *params)
+{
+	wlan_btwt_config_t btwt_config;
+	int ret = -1;
+
+	btwt_config.action = 1;
+	btwt_config.sub_id = params->btwt.sub_id;
+	btwt_config.nominal_wake = params->btwt.nominal_wake;
+	btwt_config.max_sta_support = params->btwt.max_sta_support;
+	btwt_config.twt_mantissa = params->btwt.twt_interval;
+	btwt_config.twt_offset = params->btwt.twt_offset;
+	btwt_config.twt_exponent = params->btwt.twt_exponent;
+	btwt_config.sp_gap = params->btwt.sp_gap;
+
+	ret = wlan_set_btwt_cfg(&btwt_config);
+
+	return ret;
```

Comment:
`return wlan_set_btwt_cfg(&btwt_config);` unless you want to log the return value

Reply:
Updated as comment